docs/troubleshooting/trouble-shooting.md

```diff
@@ -4,14 +4,14 @@ title: Troubleshooting
 
 ## I can't access some resources when installing Karmada
 
-- Pull images from Google Container Registry (k8s.gcr.io).
+- Pull images from Google Container Registry (registry.k8s.io).
```

Comment:
`Google Container Registry`? I think the `registry.k8s.io` now acts the agent, right?